depends/packages/openssl.mk

```diff
@@ -1,9 +1,9 @@
 # Pretty much taken verbatim from Bitcoin Core
 package=openssl
-$(package)_version=1.0.1m
+$(package)_version=1.0.1k
```

Comment:
Okay. Do you want me to accept this as-is? I'm not worried either way. I just want to be sure I understand your intentions.